Makefile-latentCrfPosTagger

```diff
@@ -3,7 +3,7 @@ SINGLE=-c
 BEFORE=-x c++ -std=c++11
 LIBS=-lfst -ldl -lboost_mpi  -lboost_program_options -lboost_serialization -lboost_thread -lboost_system -lrt
 #-llbfgs
-OPT=-O3 -DNDEBUG
+OPT=-O3 -DDEBUG -DNEURALCONST=${NEURALCONST}
```

Comment:
Please remove the additional parameter. We can read the number of dimensions of the word embeddings directly in the code rather than at compile time.